crates/frontend/src/components/table.rs

```diff
@@ -24,16 +24,19 @@ fn generate_table_row_str(row: &Value) -> String {
 #[component]
 pub fn table(
     key_column: String,
-    cell_style: String,
     columns: Vec<Column>,
     rows: Vec<Map<String, Value>>,
+    #[prop(into, default = String::new())] cell_style: String,
```

Comment:
lets add `_class` to the props which are taking classes, helps in understanding that in what format is the input expected, and as well will help in tailwind LSP